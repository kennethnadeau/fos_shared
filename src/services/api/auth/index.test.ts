import { auth } from '.'

describe('auth/service', () => {
  it('should return all required services', () => {
    expect(auth).toMatchSnapshot()
  })
})
