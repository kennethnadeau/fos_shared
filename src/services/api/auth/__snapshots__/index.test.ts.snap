// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`auth/service should return all required services 1`] = `
Object {
  "getUserInfo": [Function],
  "postAccountRegistration": [Function],
  "postLogout": [Function],
  "postOtpAuthenticate": [Function],
  "postOtpAuthenticateVerify": [Function],
  "postOtpRegistration": [Function],
  "postOtpRegistrationVerify": [Function],
}
`;
